src/main/java/org/opentripplanner/model/trafficprediction/SpeedData.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.model.trafficprediction;
+
+public class SpeedData {
+
+    private int id;
+    private long clusterId;
```

Comment:
Czy tu jest potrzebny `clusterId`?

Reply:
w pszyszlosci mmozna by to ussunac natazie mam  to poniewaz jsona generuje z zapytan sqlowych usuniencie tego wymagaloby tez zmiany generowania jsona jak juz bedziemy pewni ze wszytko sie dobrze dopasowuje puszcze komita usuwajacego to bo czasem moga sie jeszczr pojawiac niespojnosci na poziomie bazy

Reply:
okej